kit/src/elements/input/mod.rs

```diff
@@ -131,7 +131,10 @@ pub struct Props<'a> {
     aria_label: Option<String>,
     is_password: Option<bool>,
     disabled: Option<bool>,
+    #[props(optional)]
```

Comment:
why make an effort to add this back? it doesn't do anything

Reply:
Didn't know it does nothing until I read your comment on another PR the other day 😂